guides/common/modules/ref_compliance-of-rhel-hosts-in-red-hat-insights.adoc

```diff
@@ -0,0 +1,13 @@
+[id="compliance-of-rhel-hosts-in-red-hat-insights_{context}"]
+= Compliance of RHEL Hosts in Red Hat Insights
+
+For RHEL hosts with enabled Insights, you can use the Red Hat Insights compliance service to assess and monitor security policy compliance and vulnerabilities on your hosts.
+
+In this scenario, you define compliance policies in the Insights compliance service on the {RHCloud}, add your hosts to a policy, configure your hosts with the SCAP Security Guide, and scan your hosts by using the Insights client.
+You can use remote execution in {Project} to configure your hosts and scan your hosts on regular basis.
+For more information, see {ManagingHostsDocURL}configuring_and_setting_up_remote_jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
+
+[role="_additional-resources"]
+.Additional resources
+* https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html-single/assessing_and_monitoring_security_policy_compliance_of_rhel_systems[Assessing and Monitoring Security Policy Compliance of RHEL Systems] in _Red Hat Insights documentation_
```

Comment:
It looks like the d/s Insights documentation has adopted a different version numbering system. Most docs that use years like 2023 do so because they only want to support a single version. Unfortunately, Pantheon only allows versions that begin with a number, so "latest" is not possible with the current toolchain.

The Insight guides now use "1-latest". Not the prettiest solution.  :-) But it will be a lot easier to maintain all the Insights links.